Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM python:2.7
+
+ADD . /opt/ashlar
+WORKDIR /opt/ashlar
+
+RUN apt-get update
+RUN apt-get -y autoremove
+RUN apt-get install -y libgeos-dev binutils libproj-dev gdal-bin
```

Comment:
It's not strictly necessary; when I first ran `apt-get` to install the PostGIS dependencies it showed a bunch of font rendering packages as being installed but auto-removable. They're not doing any harm but it seemed odd to leave them lying around in a python image, so I decided to get rid of them. Really it should probably be done by the image maintainer.